lib/query/lib/prepareForDelivery.js

```diff
@@ -24,13 +26,31 @@ export default (node) => {
     storeOneResults(node, node.results);
 }
 
-export function applyPostFilter(node) {
-    if (node.props.$postFilter) {
-        node.results = sift(node.props.$postFilter, node.results);
+export function applyPostFilters(node) {
+    const postFilters = node.props.$postFilters || node.props.$postFilter;
+    if (postFilters) {
+        node.results = sift(postFilters, node.results);
     }
 
     node.collectionNodes.forEach(collectionNode => {
-        applyPostFilter(collectionNode);
+        applyPostFilters(collectionNode);
+    })
+}
+export function applyPostOptions(node) {
+    const options = node.props.$postOptions;
+    if (options) {
+        if(options.limit || options.skip){
```

Comment:
Yeah, sorry! :)
How about adding .eslintrc and/or .jsbeautifyrc files to the root of the project?
Btw, I normally indent using tabs (tab spacing 2) and [don't use semicolons](https://github.com/yyx990803/semi#why) :stuck_out_tongue: